src/screens/internal/ShareImage/ShareImage.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router';
+import SocialLocationPreview from 'components/SocialLocationPreview/SocialLocationPreview';
+import { Projections } from 'common/models/Projections';
+import { useProjections } from 'common/utils/model';
+import {
+  Wrapper,
+  Content,
+  ShareCardWrapper,
+  TitleWrapper,
+  LastUpdatedWrapper,
+  ScreenshotWrapper,
+} from './ShareImage.style';
+import { formatDate } from 'common/utils';
+import { findCountyByFips } from 'common/locations';
+import { Metric } from 'common/metric';
+
+/**
+ * Screen that just shows the appropriate share card so that we can take a
+ * screenshot that we then use as our OpenGraph image.
+ */
+const OpenGraphImage = () => {
+  const { stateId, countyFipsId } = useParams();
+  const isHomePage = !stateId && !countyFipsId;
+  return (
+    <Wrapper>
+      <Content>
+        <ScreenshotWrapper className={'screenshot'}>
+          <Header />
+          <ShareCardWrapper isHomePage={isHomePage}>
+            <ShareCard stateId={stateId} countyFipsId={countyFipsId} />
+          </ShareCardWrapper>
+        </ScreenshotWrapper>
+      </Content>
+    </Wrapper>
+  );
+};
+
+interface ShareCardProps {
+  stateId?: string;
+  countyFipsId?: string;
+}
+
+const Header = () => {
+  return (
+    <>
+      <TitleWrapper>Real-time COVID metrics</TitleWrapper>
+      <LastUpdatedWrapper>Updated {formatDate(new Date())}</LastUpdatedWrapper>
```

Comment:
Maybe the date should come from the data instead?

Reply:
So that would be more accurate (it'd match when the data was generated), but I decided we want this date to be as fresh as possible, especially since Twitter / Facebook cache images for ~24hrs so people are going to be seeing ~day-old images already.  So I'm cheating a bit by just using the current date.